pkg/ui/workspaces/cluster-ui/src/api/sqlApi.ts

```diff
@@ -60,6 +60,11 @@ export type SqlExecutionErrorMessage = {
   source: { file: string; line: number; function: "string" };
 };
 
+export type ApiResponse<ResultType> = {
```

Comment:
nit: just a super small preference thing, feel free to leave; I prefer `SqlApiResponse` here, it's a bit more indicative of what we're using to get the results.

Reply:
I wanna get this in, so I will update on my next PR (taking advantage of all tests that passed already :D )